doc/conf.py

```diff
@@ -51,17 +51,19 @@
 # -- Mock out some problematic modules-------------------------------------
 
 # Note that for sub-modules, all parent modules must be listed explicitly.
-MOCK_MODULES = [ 'pandas', 'pygraphviz', 'sympy',  'sympy.core', 'sympy.parsing',
-                 'sympy.parsing.sympy_parser',
-                 'sympy.printing', 'sympy.printing.mathml',
-                 'sympy.printing.lambdarepr', 'numpy',
-                 'scipy', 'scipy.integrate', 'scipy.constants', 'scipy.sparse',
-                 'h5py', 'dateutil',
-                 'dateutil.parser', 'networkx', 'networkx.algorithms',
-                 'networkx.algorithms.isomorphism',
-                 'networkx.algorithms.isomorphism.vf2userfunc',
-                 'nose', 'nose.tools', 'matplotlib', 'matplotlib.pyplot',
-                 'concurrent', 'concurrent.futures' ]
+MOCK_MODULES = ['pandas', 'pygraphviz', 'sympy',  'sympy.core',
+                'sympy.parsing', 'sympy.parsing.sympy_parser',
+                'sympy.printing', 'sympy.printing.mathml',
+                'sympy.printing.lambdarepr', 'numpy',
+                'scipy', 'scipy.integrate', 'scipy.constants', 'scipy.sparse',
+                'h5py', 'theano', 'theano.tensor', 'dateutil',
+                'dateutil.parser', 'networkx', 'networkx.algorithms',
+                'networkx.algorithms.isomorphism',
+                'networkx.algorithms.isomorphism.vf2userfunc',
+                'nose', 'nose.tools', 'matplotlib', 'matplotlib.pyplot',
+                'concurrent', 'concurrent.futures', 'pycuda.driver', 'pycuda',
+                'pyopencl']
```

Comment:
I know the formatting on this is not ideal, but please avoid reformatting it dramatically as this has introduced a merge conflict. Adding back in the extra column of spaces removes the conflict so please do that before merging master into this branch again (required for final approval). Also it would be nice to avoid rewrapping the lines you didn't edit to eliminate noise from the diff. Reformatting large blocks of code should be done in its own PR.

Reply:
Reverted back.